server/lib/realtime/application.ex

```diff
@@ -131,6 +131,10 @@ defmodule Realtime.Application do
 
     Application.put_env(:realtime, :webhook_headers, headers)
 
+    if System.get_env("PROJECT") do
+      :logger.update_primary_config(%{metadata: %{project: System.get_env("PROJECT")}})
```

Comment:
This will do a shallow merge of the logger config, overwriting entire metadata key. Should do a `get`, deep merge, then an `update`.

It will affect any other app that hooks into `:logger` config to set global metadata.

https://www.erlang.org/doc/man/logger.html#update_primary_config-1